client/my-sites/importer/importer-action-buttons/done-button.jsx

```diff
@@ -32,23 +34,38 @@ export class DoneButton extends React.PureComponent {
 
 	handleClick = () => {
 		const {
-			importerStatus: { importerId, type },
+			importerStatus: { type },
 			site: { ID: siteId },
+			siteSlug,
 		} = this.props;
+
 		const tracksType = type.endsWith( 'site-importer' ) ? type + '-wix' : type;
 
+		this.props.recordTracksEvent( 'calypso_importer_main_done_clicked', {
+			blog_id: siteId,
+			importer_id: tracksType,
+		} );
+
+		page( '/view/' + siteSlug );
```

Comment:
Is there ever chance of having no `siteSlug` after signup? I'd guess not right? If so we'd run the risk of navigating to `/view/undefined`.

Reply:
I don't think there's a risk of it being undefined in this context, but can't hurt -- will add a fallback just in case it's used elsewhere.